python/fledge/common/configuration_manager.py

```diff
@@ -294,12 +294,16 @@ def get_entry_val(k):
                     d = {entry_name: entry_val}
                     expected_item_entries.update(d)
                 num_entries = expected_item_entries.get(entry_name)
+                if num_entries is None:
+                    _logger.warning('For {} category, DISCARDING unrecognized entry name {} for item name {}'
+                                    .format(category_name, entry_name, item_name))
+                    try:
+                        del category_val_copy[item_name][entry_name]
+                    except RuntimeError:
+                            raise KeyError
```

Comment:
Position of this check should remain as the old one exists. As `if set_value_val_from_default_val and entry_name == 'value':` this should be placed before the unrecognized entry check